content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst

```diff
@@ -0,0 +1,289 @@
+Operations` pop-up window.
+
+Make changes to any of the fields in the pop-up window, and click :guilabel:`Save` once completed.
+
+Create new operations by clicking the :guilabel:`Add a line` button, and remove new operations by
+clicking the :guilabel:`Archive` button.
+
+.. note::
+   The :guilabel:`Operations` tab is *not* available by default. To enable it, navigate to
+   :menuselection:`Manufacturing app --> Configuration --> Settings`, and check the :guilabel:`Work
+   Orders` box.
+
+Once the changes are complete, navigate back to the |ECO| itself, by clicking `ECO00X` in the
+breadcrumbs in the top-left corner.
+
+On the |ECO| form, a new :guilabel:`Operation Changes` tab displays the differences between the
+current production |BOM| and the new version. Modifications to the |BOM| in an |ECO| will *not*
+affect the |BOM| used in production. That is, until the :ref:`changes are applied
+<plm/eco/apply-changes>`.
+
+.. _plm/eco/apply-changes:
+
+Apply changes
+=============
+
+After verifying the changes, move the |ECO| to a :ref:`verification stage <plm/eco/stage-config>`.
+Once the approvers accept the changes, the :guilabel:`Apply Changes` button becomes available. Click
+this button, and the |ECO| is automatically closed, and the changes are applied.
+
+To ensure the changes are live, navigate to the :guilabel:`Revision` smart button.
+
+The |BOM| will have the large red :guilabel:`Archived` tag removed. Additionally, navigate to the
+product form, switch to the :guilabel:`Miscellaneous` tab, and the :guilabel:`Version` field will be
+updated to match the version number shown on the :guilabel:`Revision` smart button of the latest
+|ECO|.
+
+.. example::
+   After applying the changes of the |ECO| for the :ref:`keyboard <plm/eco/example-keyboard>`, view
+   the version of the current keyboard |BOM| in the :guilabel:`Miscellaneous` tab. Here, the
+   :guilabel:`Version` number has been updated to `2`, matching the `V2` that appears in the
+   :guilabel:`Revision` smart button of the |ECO|.
+
+   .. image:: engineering_change_orders/BOM-version.png
+      :align: center
+      :alt: View current *BOM* version in the Miscellaneous tab.
+
+.. _plm/eco/tablet-view:
+
+Create ECO from tablet view
+===========================
+
+|ECOs| created in *tablet view* are intended for operators to directly suggest clearer operation
+instructions. To create an |ECO|, navigate to the :guilabel:`Work Orders` tab on an ongoing
+manufacturing order. Then, click the :guilabel:`tablet icon` for the desired work order to open the
+tablet view of the operation.
+
+.. image:: engineering_change_orders/tablet-icon.png
+   :align: center
+   :alt: Find the tablet icon for each operation, second from the far right.
+
+Next, add an instructional step, by clicking the :guilabel:`☰ (three horizontal lines)` icon in the
+tablet view of an operation. Then, click the :guilabel:`Add a step` button.
```

Comment:
This is another opportunity to expand on all the possible options that appear when this button is clicked.

Reply:
This is a good idea, but many of these options are related to manufacturing orders, and only the "Add a step" button is for PLM. This is a good idea to expand on these buttons in a manufacturing doc, though!

The possible options:
![image](https://github.com/odoo/documentation/assets/23611711/ff6eba9c-fe81-4b90-aaf0-102ee80eab15)